src/System.IO.FileSystem/src/System.IO.FileSystem.csproj

```diff
@@ -76,9 +76,16 @@
     <Compile Include="$(CommonPath)\System\IO\PathInternal.CaseSensitivity.cs">
       <Link>Common\System\IO\PathInternal.CaseSensitivity.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\System\Threading\Tasks\TaskToApm.cs">
+      <Link>Common\System\Threading\Tasks\TaskToApm.cs</Link>
+    </Compile>
+  </ItemGroup>
+  <!-- .NET Standard 1.7 -->
+  <ItemGroup Condition="'$(PackageTargetFramework)' == 'netstandard1.7' and '$(TargetGroup)' != 'net46' and '$(TargetGroup)' != 'net463'" >
+    <Compile Include="System\IO\FileStreamBase.NetStandard17.cs" />
```

Comment:
><Compile Include="System\IO\FileStreamBase.NetStandard17.cs" /> [](start = 4, length = 63)

@weshaggard would you rather have this as a partial class as it is, or use #ifDefs on the same FileStreamBase.cs as we are doing in other places, like System.IO.Compression.DeflateStream?